solana-programs/anchor/audius-data/cli/main.ts

```diff
@@ -1,4 +1,5 @@
-import { Program, Provider, Wallet, web3 } from "@project-serum/anchor";
+import { Program, Provider, web3 } from "@project-serum/anchor";
+import NodeWallet from "@project-serum/anchor/dist/cjs/nodewallet";
```

Comment:
why change this?

Reply:
TS complains about just Wallet it's just an interface